ldap/servers/slapd/slapi-plugin.h

```diff
@@ -6063,7 +6063,7 @@ int slapi_log_error_ext(int loglevel, char *subsystem, char *fmt, va_list varg1,
 #define SLAPI_LOG_ARGS        3
 #define SLAPI_LOG_CONNS       4
 #define SLAPI_LOG_BER         5
-#define SLAPI_LOG_FILTER      6
+#define SLAPI_LOG_FILTER      22
```

Comment:
Oh whoops. It's because setting the log level from -d on the ns-slapd binary from the cli doesn't work for me, so I have to hardcode log filter to log err to get it to show.

Reply:
Okay, resolved,